src/planscape/forsys/generate_projects_for_a_single_scenario.R

```diff
@@ -93,74 +93,36 @@ generate_projects_for_a_single_scenario <- function(forsys_input_data,
       patchmax_proj_number = 3,
       patchmax_SDW = 1,
       patchmax_EPW = 0.5,
-      patchmax_sample_frac = 0.1,
-      patchmax_exclusion_limit = 100,
+      patchmax_sample_frac = 0.01,
```

Comment:
yes: patchmax_sample_frac = 0.01 causes Patchmax to finish running much sooner (10x sooner), which is important for iterating. Depending on the constraints, it also doesn't impact accuracy much.
![image](https://user-images.githubusercontent.com/3720361/224453468-662222f9-8005-4a41-969a-d3e6a9060925.png)

patchmax_exclusion_limit does nothing right now because we don't enable thresholding. It should be a value within range [0, 1] and represents the maximum fraction of project area that can be thresholded.